src/LessMsi.Cli/eExtractionMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LessMsi.Cli
+{
+    public enum eExtractionMode
+    {
+        None = 0,
```

Comment:
You don't need to assign these numbers here these will be the default values. Do other parts of the code actually use the numeric value? Ideally other parts of the code just use the enum itself and do not rely on the numeric value anyway.

Reply:
Will amend the code accordingly.